src/yb/yql/pggate/pggate.cc

```diff
@@ -673,6 +673,8 @@ PgApiImpl::PgApiImpl(
       ybctid_reader_provider_(pg_session_),
       fk_reference_cache_(ybctid_reader_provider_, buffering_settings_, tablespace_map_),
       explicit_row_lock_buffer_(ybctid_reader_provider_, tablespace_map_) {
+  MemTracker::InitializeGcMetrics(metric_entity_);
```

Comment:
https://github.com/yugabyte/yugabyte-db/issues/17375#issue-1712663516
> This is primarily useful for the tserver, **but could also be valuable for PG,**

If I understand it correctly, we'd like to initialise metrics here as well for PG